test/e2e/kube_metrics_adapter_test.go

```diff
@@ -133,6 +133,38 @@ var _ = describe("[HPA] Horizontal pod autoscaling (scale resource: Custom Metri
 		}
 		tc.Run()
 	})
+
+    It("should scale with external metric based on hostname RPS", func() {
```

Comment:
Please add at least `[Zalando]` in the name similar to the style of other tests. This makes sure the test is in focus when we run the ginkgo test suite. Right now I think it's not being run.

Reply:
Oh, sorry I missed this detail.